prisma/schema.prisma

```diff
@@ -47,6 +47,20 @@ model Analytics {
   userId        String   @id
 }
 
+model AuthDevice {
+  createdAt           DateTime @default(now())
+  credentialId        Bytes
+  credentialPublicKey Bytes
+  counter             Int
```

Comment:
Is the counter needed?

Reply:
https://www.w3.org/TR/webauthn-2/#sctn-sign-counter